config/vlan.py

```diff
@@ -32,28 +32,53 @@ def is_dhcp_relay_running():
 
 
 @vlan.command('add')
-@click.argument('vid', metavar='<vid>', required=True, type=int)
+@click.argument('vid', metavar='<vid>', required=True)
+@click.option('-m', '--multiple', is_flag=True, help="Add Multiple Vlan(s) in Range or in Comma separated list")
```

Comment:
Instead of adding a new flag, would it be possible to extend `multiple_vlan_parser` to also be able to handle a single VID?

Reply:
multiple_vlan_parser is already able to handle single vid and multiple vid in range & comma separated list.

Reply:
If that's the case then it seems like we don't need to add the `--multiple` flag and can just pass the `vid` input to the parser directly?

Reply:
We had proposed enhancement in existing VLAN functionality for adding/deleting multiple vlan & vlan membership on a port/portchannel. This -m flag is added in support of these enhancement, keeping in view of existing vlan functionality/commands intact. All code has been done is such way as to add this support new functionality via new commands.

 It is requested to keep both behaviors (existing) and these new enhancement (proposed commands please see below  proposed commands from HLD)  as it is see user's adaptability and later we can discuss and decide based on suggestion/reviewers for code logic modifications on these proposed enhancements.


 
![Screenshot from 2024-02-07 13-04-58](https://github.com/sonic-net/sonic-utilities/assets/61490193/b7c5e923-3924-40a8-b330-55c2358d7433)